package.json

```diff
@@ -82,14 +88,15 @@
     "react-dom": "^17.0.1",
     "react-redux": "^7.2.1",
     "react-router-dom": "^5.2.0",
-    "redux": "^4.0.0"
-  },
-  "dependencies": {
+    "react-test-renderer": "17.0.1",
```

Comment:
Should this be a dependency? If it's only intended to run during local development, I believe this should be a `devDependency`?

Reply:
+1 looks like a devDep